content/post/2023/captioning-tools-extravaganza.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Captioning Tools Extravaganza!"
+date: 2023-10-09
+tags: [captions, ml]
+---
+
+In this post, I'm going to cover a wide range of automatic captioning tools based on my personal experiences with them. This is not a comprehensive list, I know there's more of them out there. For context, I'm deaf, and I'm not fluent in American Sign Language (ASL). Since 2018 when they started to become available, I've relied heavily on automatic captioning tools in my work and personal lives.
```

Comment:
Some small suggestions + highlighting that captions are really important for you because you are deaf _and_ not fluent in ASL. While that is implied, I think that spelling it out might make it more clear. 

```suggestion
In this post, I'm going to cover a wide range of automatic captioning tools based on my personal experience. This is not a comprehensive list—there are far more tools than a single person can test. For context, I'm deaf and not fluent in American Sign Language (ASL), so captioning is critical for my professional career. Since 2018, when they started to become available, I've relied heavily on automatic captioning tools in my work and personal lives.
```